package larpc;

import "../src/larpc.proto";

message TestConfig {
        // Per-node configuration. You must have one per-node configuration for
        // each node you wish to bring up in a test implementation.
        repeated NodeConfig nodes = 1;
}

message NodeConfig {
        // Machine key of this node
        required string machine_key = 1;
        required string machine_private_key = 2;
        
        repeated PrincipleDescriptor principles = 3;
}